doc/i2p.md

```diff
@@ -10,11 +10,22 @@ started with I2P terminology.
 ## Run Bitcoin Core with an I2P router (proxy)
 
 A running I2P router (proxy) with [SAM](https://geti2p.net/en/docs/api/samv3)
-enabled is required (there is an [official one](https://geti2p.net) and
-[a few alternatives](https://en.wikipedia.org/wiki/I2P#Routers)). Notice the IP
-address and port the SAM proxy is listening to; usually, it is
-`127.0.0.1:7656`. Once it is up and running with SAM enabled, use the following
-Bitcoin Core options:
+
+enabled is required. Options include:
+
+- [i2p-router](https://geti2p.net), the official implementation in Java
+- [i2pd (I2P Daemon)](https://github.com/PurpleI2P/i2pd)
+  ([documentation](https://i2pd.readthedocs.io/en/latest)), a lighter
+  alternative in C++ (successfully tested with version 2.23 and up; version 2.36
+  or later recommended)
+- [i2p-zero](https://github.com/i2p-zero/i2p-zero)
+- [other alternatives](https://en.wikipedia.org/wiki/I2P#Routers)
+
```

Comment:
For the official i2prouter implementation in Java, visit their [download page](https://geti2p.net/en/download) and follow the instructions for your Operating System.

Once installed, open a terminal window and type `i2prouter start`.  Then visit `127.0.0.1:7657` in your browser to enable SAM.  To do so, select: "Configure Homepage", then "Clients", and finally select the "Play Button" next to `SAM application Bridge`. On the left side of the page, there should be a green light next to "Shared Clients".

Reply:
If the user has successfully config'd to this point - a link will take the user to the correct page.

Maybe add a link for convenience?

Then visit [127.0.0.1:7657/configclients](http://127.0.0.1:7657/configclients)

Reply:
L09

Reply:
It's not clear to me if a change is being requested here, and what, precisely.  Install how-to seems a bit redundant with the existing docs of the various implementations.

Reply:
I think adding the [above](https://github.com/bitcoin/bitcoin/pull/22648#discussion_r684835687) at Line 23 (insert new line first) would be helpful.  I agree instructions for every implementation would be overkill, but I was completely new to i2p, and I had to do a bit of digging to figure out how to set it up.  I found the wording a bit confusing (I thought I had to install a router and install SAM).  The documentation on the website is mostly about setting up a browser.  I think adding instructions for users for the reference implementation could be helpful in increasing i2p nodes.  The tor docs include a bit of instructions as well

Reply:
Thanks. I agree it might be helpful, but I don't know if those instructions are platform agnostic. We would also need to maintain it / keep it up to date. Most people I talk to are using i2pd (admittedly perhaps because it is very easy to install), so if we explain how to install i2p-router, we should probably also do it for i2pd.  Sounds like a dedicated "How to Install an I2P Router" section.

In any case, feel free to propose this separately. I would rather not make this pull larger.